policy/datasets.yaml

```diff
@@ -166,6 +166,30 @@ deepCoadd_deblendedFlux_schema:
     storage: FitsCatalogStorage
     python: lsst.afw.table.SourceCatalog
     template: schema/deepCoadd_deblendedFlux.fits
+deepCoadd_deblendedModel:
```

Comment:
Should this be `dcrCoadd_deblendedModel` instead?

Reply:
Good catch, that was a typo.